langchain/memory/chat_message_histories/cosmos_db.py

```diff
@@ -134,11 +137,7 @@ def load_messages(self) -> None:
         except CosmosHttpResponseError:
             logger.info("no session found")
             return
-        if (
-            "messages" in item
-            and len(item["messages"]) > 0
-            and isinstance(item["messages"][0], list)
```

Comment:
out of curiosity why is this not needed?

Reply:
Because the first item in the list is not a list 😃 and as long as the list is not empty the parser can handle it!